src/components/login/login.js

```diff
@@ -72,6 +75,7 @@ class Login extends Component {
                                 </div>
                             )}
                         </Mutation>
+
```

Comment:
On this page, sign up button is submit button, so when you enter emal and password and press 'Enter', you are redirecte to signup. fix this